lib/backpex/html/resource/resource_index_table.html.heex

```diff
@@ -49,18 +49,20 @@
       </th>
       <th class="action-header-spacer" />
       <th
-        x-bind:class="hasOverflow ? 'border-gray-200 border-l' : ''"
+        x-bind:class="hasOverflow ? 'border-base-100 border-l' : ''"
         x-bind:style={"hasOverflow ? '#{shadow_sm_left()}' : ''"}
-        class="absolute right-0 rounded-tr-xl bg-gray-50 px-0"
+        class="absolute right-0 rounded-tr-xl px-0"
```

Comment:
The `cyberpunk` theme uses no border radius. Is there a generic way to solve this? I guess there are more occurrences like this (nav items, file upload, modals, ...).

![Bildschirmfoto 2024-07-04 um 17 52 40](https://github.com/naymspace/backpex/assets/133831/132098e3-617d-4577-8408-f88c6ab30a72)

Reply:
We could use cards instead of rounded, what do you think? Both of these screenshots are with card

![image](https://github.com/naymspace/backpex/assets/279956/50ebb6ee-9f56-4aee-ad96-9616494e2a39)
![image](https://github.com/naymspace/backpex/assets/279956/5a84eaf3-2512-4b00-a148-c552c9bde2ef)

Reply:
Never mind, fixed these issues using rounded utility classes from daisyui instead of the ones from tailwind, let me know if I missed something.